coalib/parsing/Globbing.py

```diff
@@ -196,7 +196,8 @@ def translate(pattern):
 
 def fnmatch(name, globs):
     """
-    Tests whether name matches one of the given globs.
+    Tests whether name matches one of the given globs. An empty glob will
```

Comment:
would be nice to add your addition starting on a new line, so the old line isnt modified.